cumulusci.yml

```diff
@@ -714,6 +714,22 @@ tasks:
                 u.LanguageLocaleKey = languageMap.get('%%%PARAM_1%%%');
                 Update u;
 
+    remove_feature_parameters_from_XML:
+        class_path: cumulusci.tasks.metadata.modify.RemoveElementsXPath
+        options:
+            output_style: salesforce
+            elements:
+                - path: src/package.xml
+                  xpath: /ns:Package/ns:types/ns:members
+                            [../ns:name='FeatureParameterBoolean' or ../ns:name='FeatureParameterInteger']
+                            [.='PilotEnabled' or
```

Comment:
I'm pretty sure these two won't help with the LDV-XL org. The best one to delete is `Data_CountGERowsLast30DaysBatch`. That leaves exactly 25 feature parameters. Any org that already has those 25 (such as LDV-XL) will require that the most recent ones (`Data_CountGERowsLast30DaysBatch`) be deleted. Once it exists in the, removing from the metadata does not remove it from the org during deployment. When new FP's are added, then we can add those to this task.